.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -8,37 +11,38 @@ body:
         - Prior to submitting, kindly search the closed issues to check if the problem you are reporting has already been addressed and resolved. If you come across a closed issue that pertains to your problem, please leave a comment on that issue instead of creating a new one.
         - If the default Linux distribution is not adhered to, script support will be discontinued.
         - When encountering the error message `[ERROR] in line 23: exit code *: while executing command "$@" > /dev/null 2>&1`, make sure to run the script in verbose mode to accurately determine the underlying issue.
-        - For suggestions, questions or feature/script requests, please share them in the [Discussions section.](https://github.com/community-scripts/ProxmoxVE/discussions)
+        - For suggestions, questions, or feature/script requests, please use the [Discussions section.](https://github.com/community-scripts/ProxmoxVE/discussions)
+
   - type: input
     id: guidelines
     attributes:
       label: Please verify that you have read and understood the guidelines.
-      placeholder: 'yes'
+      placeholder: "yes"
     validations:
       required: true
+
   - type: textarea
     id: bug
     attributes:
       label: A clear and concise description of the issue.
     validations:
       required: true
-  - type: dropdown
-    id: settings
+      
+  - type: checkboxes
```

Comment:
Why change this to a multi-select?

Reply:
looks better and its possible to check both if both try